mandatory/2-function-creation.js

```diff
@@ -15,26 +21,69 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200){
+    newPrice = totalPrice - (0.1 * totalPrice);
+
+  }
+  else{
+    newPrice = totalPrice - (0.05 * totalPrice);
+  }
+ return newPrice;
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  number = 0;
+  while(number < limit){
+    number = number + 1;
+    if (number === 1){
+    console.log(number);
+  }else if(number % 2 !== 0){
+    console.log(number)
+  }
+    
+
+  }
+}
```

Comment:
Nice. This works but we can make it even simpler because `1%2` is equal to 1. So it will go into your if statement.
```
function printOddNumbers(limit) {
  number = 0;
  while(number < limit){
    number = number + 1;
    if(number % 2 !== 0) {
      console.log(number)
    }
  }
}